src/core/lib/load_balancing/lb_policy.cc

```diff
@@ -72,15 +72,16 @@ LoadBalancingPolicy::PickResult LoadBalancingPolicy::QueuePicker::Pick(
   //    ExitIdleLocked().
   if (!exit_idle_called_ && parent_ != nullptr) {
     exit_idle_called_ = true;
-    auto* parent = parent_->Ref().release();  // ref held by lambda.
+    auto* parent = parent_->Ref(DEBUG_LOCATION, "QueuePicker")
```

Comment:
Suggest moving this change to a separate PR.

Note that we can probably change the `ExecCtx::Run()` below to instead use `EventEngine::Run()`, and then we can capture the ref in the lambda, just like we're doing everywhere else.

Reply:
Removed. It was helpful in debugging, I thought it would be harmless to leave in, but I'll file a separate PR.

Reply:
https://github.com/grpc/grpc/pull/31836